src/lightning_app/runners/cloud.py

```diff
@@ -41,6 +48,9 @@
 from lightning_app.utilities.packaging.app_config import AppConfig, find_config_file
 from lightning_app.utilities.packaging.lightning_utils import _prepare_lightning_wheels_and_requirements
 
+if TYPE_CHECKING:
+    pass
```

Comment:
hehe that's funny :)

Reply:
uhhh... :sweat_smile: woops? 

(fixed this :joy:)